chaostoolkit/data/journal.json

```diff
@@ -0,0 +1,159 @@
+{
```

Comment:
We don't need this file to be checked-in , it is generated file. Will add it in ignore.